src/main/java/it/uniroma2/pellegrini/z64sim/controller/SettingsController.java

```diff
@@ -186,9 +192,10 @@ private static class Settings {
         private String logLevel;
         private boolean logShowDateTime;
         private String logOutFile;
-        private List<String> openFiles;
+        private List<String> openFiles; // TODO: unused setting
```

Comment:
I would remove this for now. The editor is undergoing a major refactor that could eventually support multiple files. But this will also require a different strategy for program loading.

Reply:
Ok, sure